test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/ModelBinderFactoryTest.cs

```diff
@@ -235,6 +236,109 @@ public void CreateBinder_Caches_WhenTokenIsNotNull()
             Assert.Same(result1, result2);
         }
 
+        public static TheoryData BindingInfoData
```

Comment:
Aside from the type that's being inspected this data is identical to the widget in your other test class. Could share data by either making a generic data builder method or both expecting the same types.